icons/check-check.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M18 7 7 18l-5-5" />
+  <path d="m22 11-7.5 7.5L13 17" />
```

Comment:
These are 1px off to the bottom compared to `check`, I guess that probably wasn't intentional.
```suggestion
  <path d="M18 6 7 17l-5-5" />
  <path d="m22 10-7.5 7.5L13 16" />
```